node/pkg/publicrpc/publicrpcserver.go

```diff
@@ -68,6 +68,13 @@ func (s *PublicrpcServer) GetSignedVAA(ctx context.Context, req *publicrpcv1.Get
 		return nil, status.Error(codes.InvalidArgument, "no message ID specified")
 	}
 
+	chainID := vaa.ChainID(req.MessageId.EmitterChain.Number())
+
+	// This interface is not supported for PythNet messages because those VAAs are not stored in the database.
```

Comment:
Do we really need to treat this specially?  Getting the signed VAA for pythnet will return a "VAA not found" error, which seems fine?